liwords-ui/src/lobby/active_games.tsx

```diff
@@ -17,12 +24,44 @@ type Props = {
 };
 
 export const ActiveGames = (props: Props) => {
+  const { useState } = useMountedState();
+  const [sortByLexicon, setSortByLexicon] = useState(
+    localStorage.getItem('sortByLexicon')
+  );
   const navigate = useNavigate();
   const {
     loginState: { perms },
   } = useLoginStateStoreContext();
   const isAdmin = perms.includes('adm');
 
+  const handleChange = useCallback(
+    (
+      pagination: TablePaginationConfig,
+      filters: Record<string, FilterValue | null>,
+      sorter:
+        | SorterResult<ActiveGameTableData>
+        | SorterResult<ActiveGameTableData>[],
+      extra: TableCurrentDataSource<ActiveGameTableData>
+    ) => {
+      if (extra.action === 'filter') {
+        if (filters.lexicon) {
+          if (filters.lexicon.length === 1) {
+            const lexicon = filters.lexicon[0] as string;
+            if (lexicon !== sortByLexicon) {
+              setSortByLexicon(lexicon);
+              localStorage.setItem('sortByLexicon', lexicon);
+            }
+          }
+        } else {
+          // filter is reset, remove lexicon
+          setSortByLexicon(null);
+          localStorage.removeItem('sortByLexicon');
+        }
+      }
+    },
+    []
```

Comment:
this requires a dependency `sortByLexicon` in this array, since if that changes, it should change the callback. The `docker-compose` logs for the `frontend` should show this.

Reply:
thanks, I'll add it, I haven't used react since 2017, so much has changed. I need to read about this